### ISSUE TITLE
Change in download directory not taking effect after restart

### ISSUE THREAD

**Issue by masonc15** [mrichards42/xword#116]
Hello,

When I change the preferred download directory from the default, it takes effect for the remainder of that solving session.  But as soon as I close the Xword app and relaunch it, it goes back to the default.  This is frustrating and I'm not sure why this is happening.  Let me know if there is any more information I can provide.  I am running version 0.7.0.

**Comment by mrichards42** [comment]
Good catch, thanks! Looks like the download config isn't saving at all 🤦. I'll have a fix up for that in the next few days, but in the mean time you should be able to work around it by creating the directory that it expects to save the config to.

If you're on a mac, that's

`/Users/{username}/Library/Application Support/XWord/config/download/`

Or if you're on windows,

`C:\Users\{username}\AppData\Roaming\XWord\config\download`

**Comment by masonc15** [comment]
Thanks for the response!  Looks like creating that directory worked for now.
